### PR TITLE
MODUSERS-388 - Adding DCB as a user type.

### PR DESCRIPTION
**Pull request by gurleenkaurbp**
The purpose of this PR is to add a new user type dcb and also added missing pull_request_template.md file.
No breaking change.

### CODE REVIEW

**Review by SinghAdes — APPROVED** [review]

**Review by Vignesh-kalyanasundaram — APPROVED** [review]

**Review by pavankumar181 — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+ The goal is not only to explain what you did, but help other
+ developers *work* with your solution in the future.
+-->
+
+#### TODOS and Open Questions
+<!-- OPTIONAL
+- [ ] Use GitHub checklists. When solved, check the box and explain the answer.
+-->
+- [ ] Check logging
```
Comment (azizbekxm) [inline PULL_REQUEST_TEMPLATE.md:42]: that was inside of the comment in most pr template I have seen.

Comment (gurleenkaurbp) [reply PULL_REQUEST_TEMPLATE.md:42]: are you suggesting to remove this line ?

Comment (azizbekxm) [reply PULL_REQUEST_TEMPLATE.md:42]: no, move to inside of comment

Comment (gurleenkaurbp) [reply PULL_REQUEST_TEMPLATE.md:42]: oh ok will do thanks

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**